lib/runners/hook_runner.js

```diff
@@ -140,14 +130,25 @@ HookRunner.prototype = {
       thing = data ? template(thing, data) : thing;
       return thing;
     }
-  },
-  stop: function() {
+  }
+
+  stop() {
     if (this.hookProcess) {
       return this.hookProcess.kill();
     }
 
     return Bluebird.resolve();
   }
+}
+
+HookRunner.with = (config, hook, data) => {
```

Comment:
this can become `static with(config, hook, data) { /* ... */ }`

Reply:
ok ty sir

Reply:
Done